docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,244 @@
+    sudo systemctl start nginx
+    sudo systemctl enable nginx
+    sudo setsebool -P httpd_can_network_connect 1
+    sudo firewall-cmd --permanent --zone=public --add-service=http
+    sudo firewall-cmd --permanent --zone=public --add-service=https
+    sudo firewall-cmd --reload
+
+    # Generate private/public keys (optional if you do not already have SSL certificates):
+    sudo mkdir /etc/carta
+    cd /etc/carta
+    sudo openssl genrsa -out carta_private.pem 4096
+    sudo openssl rsa -in carta_private.pem -outform PEM -pubout -out carta_public.pem
+
+    # Set up the nginx configuration file using our sample configuration file linked below:
+    sudo cd /etc/nginx/conf.d/
+    sudo vi /etc/nginx/conf.d/carta.conf
+    sudo systemctl restart nginx
+
+    # Check it is running:
+    sudo systemctl status nginx
+
+A :ref:`sample configuration file<example_nginx>` is provided in the configuration section. This should be adapted to your server configuration.
+
+.. note::
+    If there are problems, you can debug with ``journactl -xe`` and checking log files in ``/var/log/nginx/``.
+
+
+6. Create the 'carta' user
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For security, we recommend not to run the carta-controller as the root user. Therefore we create a new user called ``carta`` and make part it part of a new group called ``carta-users``. 
+We will allow any user in the ``carta-users`` group to run ``/usr/bin/carta_backend`` and the script to close the carta-backend; ``/usr/local/bin/carta-kill-script`` by adding a custom entry to the ``sudoers`` file.
+
+.. code-block:: shell
+    
+    # Create the carta user:
+    sudo adduser carta
+    sudo groupadd carta-users
+    sudo usermod -a -G carta-users carta
+    # Check everything is OK
+    id carta
+    # It should show 'uid=1000(carta) gid=1000(carta) groups=1000(carta),1001(carta-users)'
+
+    # So that log files can be written:
+    sudo mkdir -p /var/log/carta
+    sudo chown -R carta /var/log/carta
+
+    # Add the custom sudoers file entry using our sample linked below
+    sudo visudo -f /etc/sudoers.d/carta_controller
+    
+An :ref:`example sudoers configuration<example_sudoers>` is provided in the configuration section.
+
+.. note::
+    The only safe way to modify sudoers is using ``visudo``. Any syntax errors from directly editing sudoers could make your system unusable.
+
+
+7. Set up the user authentication method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the most difficult step and depends how you authenticate users at your institute. 
+In this step-by-step guide we use PAM authntication and local user, ``bob``, on the server running carta-controller.
+Every user needs to be part of the ``carta-users`` group.
+With PAM authentication, the ``carta`` user that runs carta-controller requires access to the ``/etc/shadow`` file in order to authenticate other users. We can enable this by creating a new group called ``shadow`` and assigning the ``/etc/shadow`` file to that group:
```

Comment:
I think it's worth clarifying that you only need `/etc/shadow` access when using PAM with local authentication. PAM using LDAP doesn't require this.